modules/core/04-channel/types/msgs_test.go

```diff
@@ -840,7 +826,7 @@ func (suite *TypesTestSuite) TestMsgChannelUpgradeOpenValidateBasic() {
 		suite.Run(tc.name, func() {
 			msg = types.NewMsgChannelUpgradeOpen(
 				ibctesting.MockPort, ibctesting.FirstChannelID,
-				types.OPEN, suite.proof,
+				types.STATE_FLUSHCOMPLETE, suite.proof,
```

Comment:
should we have another success case where state is `OPEN`?

Reply:
we certainly should!